internal/pkg/skuba/deployments/ssh/kubernetes.go

```diff
@@ -39,7 +39,9 @@ const (
 func init() {
 	stateMap["kubernetes.bootstrap.upload-secrets"] = kubernetesUploadSecrets(KubernetesUploadSecretsContinueOnError)
 	stateMap["kubernetes.join.upload-secrets"] = kubernetesUploadSecrets(KubernetesUploadSecretsFailOnError)
-	stateMap["kubernetes.install-node-pattern"] = kubernetesInstallNodePattern
+	stateMap["kubernetes.install-fresh-pkgs"] = kubernetesFreshInstallAllPkgs
+	stateMap["kubernetes.upgrade-kubeadm-pkg"] = kubernetesUpgradeKubeadmPkg
+	stateMap["kubernetes.upgrade-other-kubernetes-pkgs"] = kubernetesUpgradeOtherPkgs
```

Comment:
Maybe we can come up with a better name for `upgrade-other-kubernetes-pkgs`. Maybe we can have a `upgrade-k8s-stage-1` (which does `kubeadm`) and a `upgrade-k8s-stage-2`

Reply:
Naming is hard :)

Anything goes. Is stage1/stage2 more explicit than kubeadm/the rest? The rest is definitely _not_ clear, but stages are also hard... I am fine with changing to stage1/stage2 though.

Reply:
I have updated to stage 1 and stage 2. Not sure if it's more explicit, but we can refactor later if necessary, if we find a better name :laughing: